policyengine_us/parameters/gov/states/or/oha/healthier_oregon/eligibility/adult_income_limit.yaml

```
@@ -0,0 +1,15 @@
+description: Oregon Healthier Oregon sets this income limit for adults as a percentage of FPL
+metadata:
+  unit: /1
+  period: year
+  label: OR Healthier Oregon adult income limit (% FPL)
```

```suggestion
  label: Oregon Healthier Oregon adult income limit (% FPL)
```
And other label in parameter.